api/lib/domain/usecases/index.js

```diff
@@ -141,6 +141,7 @@ const dependencies = {
   userEmailRepository: require('../../infrastructure/repositories/user-email-repository'),
   userOrgaSettingsRepository: require('../../infrastructure/repositories/user-orga-settings-repository'),
   userReconciliationService: require('../services/user-reconciliation-service'),
+  userCreationRepository: require('../../infrastructure/repositories/user-creation-repository'),
```

Comment:
```suggestion
  userToCreateRepository: require('../../infrastructure/repositories/user-to-create-repository'),
```

Mais en fait je me pose la question de savoir si c'est pertinent d'avoir un repo dédié uniquement à la création d'utilisateur (sans être sûr de la bonne façon de faire). Est-ce qu'on a une règle qui dit que chaque modèle doit avoir son repo ?

Reply:
En fait vu le nombre de méthode dans `user-repo` je me suis dis que je risquais de mettre trop de bazars si je voulais y rajouter la méthode de création d'utilisateur dedans. 
Dans l'idéal `user-to-create-repository` pourrait servir à tout ce qui touche à la création d'un utilisateur : sa création dans la table `user` , en passant par l'ajout de sa méthode de connexion voir aux envois d'email si besoin. Genre tout le pan fonctionnel qui touche à la création d'un user.
Le hic c'est que j'ai pas eu la foi de refacto plus loin et que je ne sais pas si mon intention est bonne 😅